scale/docs/rest/v6/strike.rst

```diff
@@ -0,0 +1,496 @@
+            "files_to_ingest": [{ 
+                "filename_regex": ".*txt" 
+            }] 
+        } 
+    } 
+    
++-------------------------------------------------------------------------------------------------------------------------+
+| **Edit Strike**                                                                                                         |
++=========================================================================================================================+
+| Edits an existing Strike process with associated configuration                                                          |
++-------------------------------------------------------------------------------------------------------------------------+
+| **PATCH** /v6/strikes/{id}/                                                                                             |
+|           Where {id} is the unique identifier of an existing model.                                                     |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **Content Type**   | *application/json*                                                                                 |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **JSON Fields**                                                                                                         |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| title              | String            | Optional | The human readable display name of the Strike process.              |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| description        | String            | Optional | A longer description of the Strike process.                         |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| configuration      | JSON Object       | Optional | JSON defining the Strike configuration.                             |
+|                    |                   |          | (See :ref:`architecture_strike_spec`)                               |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| **Successful Response**                                                                                                 |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **Status**         | 200 OK                                                                                             |
++--------------------+----------------------------------------------------------------------------------------------------+
```

Comment:
Change edit API to return 204 No Content. We'll standardize on that for all "edit" API calls.